index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-innercontainer">
+            <span class="nav-logo">My Logo</span>
+            <ul class="nav-items">
+                <li class="nav-link">Portfolio</li>
+                <li class="nav-link">About</li>
+                <li class="nav-link">Contact</li>
+                <li class="nav-link"><img src="./Assets/logos/envelope.png" class="icon" alt="" /></li>
+            </ul>
+            <div class="baricon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </div>
+    </nav>
+    <section class="herosection">
+        <div class="textcontainer">
+            <h5 class="h5">Hey There I’m Life.</h5>
```

Comment:
- Kindly change this tag `h5` for an `h1` tag, remember to use `h1` tags for titles and h onforward tags for subtitles or another type of information on your website. This will help you to make a better semantic file and follow the best practices for accessibility and screen readers for example 🥇